cli/replicaset/cmd/upgrade.go

```diff
@@ -0,0 +1,272 @@
+package replicasetcmd
+
+import (
+	_ "embed"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/tarantool/tt/cli/connector"
+	"github.com/tarantool/tt/cli/replicaset"
+	"github.com/tarantool/tt/cli/running"
+)
+
+var (
+	// List of replicaset names specified by the user for the upgrade.
+	ChosenReplicasetAliases []string
+	// Timeout period (in seconds) for waiting on LSN synchronization.
+	LsnTimeout int
+)
+
+type instanceMeta struct {
+	run  running.InstanceCtx
+	conn connector.Connector
+}
+
+//go:embed lua/upgrade.lua
+var upgradeMasterLua string
+
+type syncInfo struct {
+	LSN uint64  `mapstructure:"lsn"`
+	IID uint32  `mapstructure:"iid"`
+	Err *string `mapstructure:"err"`
+}
+
+// Filters the given replicaset list by chosen aliases and returns the chosen
```

Comment:
It is an idiomatic way to start a comment.
```suggestion
// filterReplicasetsByAlieases filters the given replicaset list by chosen aliases and returns the chosen
```

Reply:
Fixed.